tests/api/switch_api_test.py

```diff
@@ -320,7 +320,7 @@ def test_anonymous_switch_can_be_netman_proxied(self):
             username='root',
             password='password',
             port=None,
-            netman_server='1.2.3.4').once().ordered().and_return(self.switch_mock)
+            netman_server=['1.2.3.4']).once().ordered().and_return(self.switch_mock)
```

Comment:
Yup, but the only known use for now is our config file and it is being updated, we could still honor the old string parameter but the code would need a type check which I'm not a fan of, what do you think?